crates/subspace-networking/src/constructor.rs

```diff
@@ -68,14 +68,18 @@ const MAX_CONCURRENT_STREAMS_PER_CONNECTION: usize = 10;
 const ENABLE_GOSSIP_PROTOCOL: bool = false;
 
 const TEMPORARY_BANS_CACHE_SIZE: u32 = 10_000;
-const TEMPORARY_BANS_DEFAULT_BACKOFF_INITIAL_INTERVAL: Duration = Duration::from_secs(5);
+/// The initial temporary ban delay, before attempting a reconnection.
+/// This is also effectively the fixed reconnection delay for reserved peers disconnected by the
+/// remote end.
+const TEMPORARY_BANS_DEFAULT_BACKOFF_INITIAL_INTERVAL: Duration = Duration::from_secs(10);
```

Comment:
IIRC temporary bans are actually deactivated in the code, so this probably makes no difference